ports/atmel-samd/boards/xinabox_cs11/mpconfigboard.mk

```diff
@@ -26,5 +26,4 @@ CIRCUITPY_RTC=0
 CIRCUITPY_COUNTIO=0
 
 # Include these Python libraries in firmware.
-FROZEN_MPY_DIRS += $(TOP)/frozen/Adafruit_CircuitPython_BusDevice
```

Comment:
I think you'll actually want to enable CIRCUITPY_BUSDEVICE here because it isn't a full build.

Reply:
Did another sweep, missed some were frozen but not full builds so think I caught them all.